### DISCUSSION TITLE
Failed to start kernel

### DISCUSSION THREAD

**Discussion by Nand-creator** [AammarTufail/python_ka_chilla2023#24] [Q&A]
Hello, Sir i am unable to run ipy file, because it is showing the kernel error, i have attached the screenshot below please check and response me to solve this issue.
![Capture](https://user-images.githubusercontent.com/121835093/211036932-e4c7e7c5-d2fd-400d-adac-d2b28117bda1.PNG)

**Comment by SahilJatoi744** [comment]
Ap VS code ko restart kary aur python ka version select kary from right corner

**Comment by Nand-creator** [comment]
![Uploading Capture1.PNG…]()
I am not getting you, their is no such any option, please check the
screenshot below.

**Reply by SahilJatoi744** [reply]
show me full screen shot of your VS code
it is on top right of vs code

**Comment by Masoodullah909** [comment]
> Hello, Sir i am unable to run ipy file, because it is showing the kernel error, i have attached the screenshot below please check and response me to solve this issue. ![Capture](https://user-images.githubusercontent.com/121835093/211036932-e4c7e7c5-d2fd-400d-adac-d2b28117bda1.PNG)

![image](https://user-images.githubusercontent.com/109905704/211074502-e351afb4-254a-4c58-aba7-e3eec08088ad.png)
1.  You Can Select Your Kernel 
2.  If the kernel is Corrupted or not Working You may restart the kernel.
3.  If all the above not working you may restart your VSCode.

**Comment by Nand-creator** [comment]
I don't have kernal option at the top right corner, and the restart option
is also not available, I can not able to use the restart option.

I have also restarted my VS Code many times but it still doesn't work.

**Reply by SahilJatoi744** [reply]
apne python install kiya hai
uska path set kiya hai

**Reply by Hashirahmad996** [reply]
apny  .ipynb file ni bnai to option ni ay ga.

**Comment by Nand-creator** [comment]
Haan path set kiya tha

**Reply by SahilJatoi744** [reply]
I think version ka masla hai ap dosra python ka version install kary aur usse jab install kary phr set path pe tick karna aur phr jab install hojai
phr ap vs code ko band karke open karna set hojaiga

**Reply by SahilJatoi744** [reply]
VS code me python ke extension hoti hai wo apni ke hai

**Comment by Nand-creator** [comment]
Haan wo bhi kiya hy

**Reply by SahilJatoi744** [reply]
acha phr python installa karle phr se

**Reply by SahilJatoi744** [reply]
![image](https://user-images.githubusercontent.com/89636837/211081498-1e3370f2-6692-40c2-b7ef-9e19779d777a.png)
ye sari extension karle

**Comment by Nand-creator** [comment]
Bhai kr liya phr se install aur jo ap ne bola wo sub bhi, phr b same error
aaraha hy

**Reply by Hashirahmad996** [reply]
puri screen ka screen shot send krein error k sath

**Comment by Nand-creator** [comment]
Bhai ap ne jo kaha k up right corner m kernel select kren, wo nh aaraha
mery pas.

Mery pas bs python e aaraha hy wahan, search bhi kiya usy select kr k phr
bhi nh aaraha kernel ka option.


Please guide kr den k wahan kernel ksy add krun?

**Reply by SahilJatoi744** [reply]
![image](https://user-images.githubusercontent.com/89636837/211166861-7200ce88-ba03-4aa3-b505-13e562f2da20.png)

**Reply by bakhtawar-ishaq** [reply]
aoa, mera bhi yahi issue a raha ha. apka resolve hua?

**Reply by bakhtawar-ishaq** [reply]
Hello there, mera problem solve ho gaya ha to socha k apko bta dn shayd it might help you as well. ap python ko unistall krein or dobara download krein. jab ap first time usko run krein ge to wahan 2 tick walay option ain ge, unmein aik hoga "allow path.exe" aesa kuch likha a raha hoga us ko lazmi check krein phir istall k button pe click krein.
